aptos-move/framework/aptos-framework/sources/aggregator_v2/aggregator_v2.move

```diff
@@ -12,12 +12,6 @@
 /// at the moment.**
 module aptos_framework::aggregator_v2 {
 
-    /// The value of aggregator overflows. Raised by native code.
-    const EAGGREGATOR_OVERFLOW: u64 = 1;
-
-    /// The value of aggregator underflows (goes below zero). Raised by native code.
-    const EAGGREGATOR_UNDERFLOW: u64 = 2;
-
     /// Aggregator feature is not supported. Raised by native code.
     const ENOT_SUPPORTED: u64 = 3;
```

Comment:
I think we can delete this one, it is not used anywhere? (also in the original aggregator it seems)